compiler/src/iree/compiler/ConstEval/JitGlobals.cpp

```diff
@@ -831,6 +882,10 @@ class JitGlobalsPass final : public impl::JitGlobalsPassBase<JitGlobalsPass> {
       programBuilder.getTargetModule()->erase();
       return;
     }
+    if (failed(stripExecutionContextAffinities(
+            programBuilder.getTargetModule()))) {
+      return signalPassFailure();
+    }
```

Comment:
Seems like this code could instead go in `importInitializer()`. Do we have a preference for making the import function take on more responsibility or treating this as a cleanup step?

Reply:
I moved it there.